### PR TITLE
feat: add email subscription modal

### PR DESCRIPTION
**Pull request by eliobricenov**
<--- Put the description here --->

The modal will show after the user finishes the onboarding.

How to test:
- import a MM wallet or connect MM mobile or wallet connect

---

Use the following Checklist if you have changed something on the Backend or Frontend:

- [ ] Tested the feature and it's working on a current and clean install.
- [ ] Tested the main App features and they are still working on a current and clean install. (Login, Install, Play, Uninstall, Move games, etc.)
- [ ] Created / Updated Tests (If necessary)
- [ ] Created / Updated documentation (If necessary)

### CODE REVIEW

**Review by BrettCleary — APPROVED** [review]

File: src/frontend/screens/Onboarding/walletSelection/index.tsx [hunk 1]
```diff
@@ -144,6 +145,7 @@ const WalletSelection: React.FC<WalletSelectionProps> = function (props) {
     dbPath?: string,
     browser?: ImportableBrowser
   ) {
+    authState.enableOpenAuthModalOnAppReload()
```
Comment (eliobricenov) [inline src/frontend/screens/Onboarding/walletSelection/index.tsx:148]: enable user imports MM and reloads the app

File: src/frontend/screens/Onboarding/walletSelection/index.tsx [hunk 2]
```diff
@@ -165,6 +167,7 @@ const WalletSelection: React.FC<WalletSelectionProps> = function (props) {
     })
     wait(4000).then(() => {
       props.disableOnboarding('connected')
+      authState.openSignInModal()
```
Comment (eliobricenov) [inline src/frontend/screens/Onboarding/walletSelection/index.tsx:170]: open after user finishes onboarding

**Review by red-game-dev — APPROVED** [review]

**Comment by eliobricenov** [comment]
A good portion of the component is similar to the `AuthModal` component but I wanted to decouple them so we have more control over it. By using a different electron partition we also make sure that we get the same layout even if the user logs in, which is not the case if we use the `AuthModal`.

**Review by nyghtstalker — APPROVED** [review]
Works great!

**Comment by BrettCleary** [comment]
works great for me

let's add an .env.example file with the LD flag

**Comment by BrettCleary** [comment]
Tested again and it works great for me!

**Pull request merged**